src/timeline/js/functions.js

```diff
@@ -298,10 +298,12 @@ function moveBoundingBox(scope, previous_x, previous_y, x_offset, y_offset, left
   }
 
   // Find closest nearby object, if any (for snapping)
-  var bounding_box_padding = 3; // not sure why this is needed, but it helps line everything up
-  var results = scope.getNearbyPosition([bounding_box.left, bounding_box.right + bounding_box_padding], 10.0, bounding_box.selected_ids);
-  var nearby_offset = results[0];
-  var snapline_position = results[1];
+    var [nearby_offset, snapline_position]
+      = scope.getNearbyPosition(
+        [bounding_box.left, bounding_box.right],
+        10.0, bounding_box.selected_ids);
+  // var nearby_offset = results[0];
+  // var snapline_position = results[1];
```

Comment:
Heh, forgot I even did that. Our JS code creates _way_ too many temporary variables, as it's tossing around data. This eliminates one temporary array, which is something.